ulib/Makefile

```diff
@@ -1,8 +1,8 @@
 .PHONY: all
 
 FSTAR_HOME=..
-include gmake/z3.mk
-include gmake/fstar.mk
+# include gmake/z3.mk
+# include gmake/fstar.mk
```

Comment:
Removed a double inclusion of fstar.mk z3.mk (also included in Makefile.verify etc.) 

Should probably just delete these lines